erpnext/assets/doctype/asset_value_adjustment/asset_value_adjustment.py

```diff
@@ -76,7 +77,8 @@ def validate_date(self):
 			)
 
 	def set_difference_amount(self):
-		self.difference_amount = flt(self.current_asset_value - self.new_asset_value)
+		self.difference_amount = flt(self.new_asset_value - self.current_asset_value)
```

Comment:
As we are reversing the sign of difference_amount value (which is correct), we need to write a patch to update the value of existing asset value adjustment records.

You can refer https://github.com/frappe/erpnext/pull/40390/files#diff-c3d346867df08838dc46256708b1cce71ed729e35ff7032e4eef16b752242a80